collector/src/toolchain.rs

```diff
@@ -372,13 +372,13 @@ pub fn get_local_toolchain(
     // specified. This follows the similar pattern used by rustup's binaries
     // (e.g., `rustc +stage1`).
     let (rustc, id) = if let Some(toolchain) = rustc.strip_prefix('+') {
-        let output = Command::new("rustup")
+        let status = Command::new("rustup")
```

Comment:
Hmm, this will now randomly print the path to the toolchain to stdout, right? By default, `Command` inherits stdout and stderr, so if you use `status`, it will just print its output. We could add
```rust
.stdout(Stdio::null())
.stderr(Stdio::null())
```
although I personally just use `output` if I want to "hide it" and I don't expect it to be large.